blocks/identity-block/components/EditableFormInputField/index.jsx

```diff
@@ -0,0 +1,72 @@
+import { Button, BUTTON_STYLES, BUTTON_TYPES } from '@wpmedia/shared-styles';
+import React, { useState } from 'react';
+import FormInputField, { FIELD_TYPES } from '../FormInputField';
+
+import './styles.scss';
+
+function EditableFieldPresentational({
+  initialValue,
+  error,
+  label,
+  validationMessage,
+}) {
+  const [isEditable, setIsEditable] = useState(false);
+  return (
+    <section className="editable-form-input">
+      <div className="editable-form-input--internal">
+        {
+        isEditable ? (
+          <>
+            <FormInputField
+              type={FIELD_TYPES.EMAIL}
+              label={label}
+              defaultValue={initialValue}
+              showDefaultError={false}
+              required
+              autoComplete="email"
+              name="email"
+              validationErrorMessage={validationMessage}
+            />
```

Comment:
this input would be the child. keep the save and cancel and other buttons. and, as long as everything is wrapped in a form, this will submit

Reply:
updated